### DISCUSSION TITLE
HDwallet from public key?

### DISCUSSION THREAD

**Discussion by Idan-Shperling** [trustwallet/wallet-core#2383] [Q&A]
If I create an EC private and public key pair, can I create HDwallet from that public key?

**Comment by hewigovens** [comment]
No, just `PrivateKey` and `PublicKey` classes

**Reply by Idan-Shperling** [reply]
thanks. and what about from xpubkey?

**Reply by hewigovens** [reply]
No, same like `HDWallet`
